src/pybind/mgr/mgr_util.py

```diff
@@ -29,6 +39,228 @@
 logger = logging.getLogger(__name__)
 
 
+class CephfsConnectionException(Exception):
+    def __init__(self, error_code, error_message):
+        self.errno = error_code
+        self.error_str = error_message
+
+    def to_tuple(self):
+        return self.errno, "", self.error_str
+
+    def __str__(self):
+        return "{0} ({1})".format(self.errno, self.error_str)
+
+
+class ConnectionPool(object):
```

Comment:
```suggestion
class CephfsConnectionPool(object):
```
? We're still in `mgr/mgr_util.py`.